frontend/quickcheck/src/pages/ProjectOverview.jsx

```diff
@@ -8,17 +8,12 @@ import { Link } from 'react-router-dom';
 
```

Comment:
These Changes break the Add project button, i think this is an old version of this file.

Reply:
How should this be resolved?

Reply:
`git checkout origin/development ProjectOverview.jsx` should do the trick. It resets the file to the state on development

Reply:
Ok, tried to do this. Please let me know if it works now, when you have time :)